lib/Core/canBeDeleted.js

```diff
@@ -0,0 +1,10 @@
+function canBeDeleted(catalogMember) {
```

Comment:
This should go in `Models` not `Core`. Also, is it needed? I think on the top-level groups and items that you create on the My Data tab, `removable` is true, and for any nested items and groups inside it `removable` is false. No need to check at every level.

Reply:
removed `canBeDeleted.js`, i see that's not needed :)